### PR TITLE
Adds a syndicate mining hardsuit available to the free miners.

### PR DESCRIPTION
**Pull request by nesisknesis**
Exemplifies that the free miners are not nanotrasen employed and are an independent faction. 

The Blood-red design has been replaced with a brown/tan colour scheme, and the dual mode function has been removed with the functions of a normal mining suit taking its place. The green helmet light remains though.

image
image

image
image

The mining\eva suit storage unit which is located on the free miner ship and the asteroid have been tweaked to have the syndicate mining hardsuit instead of the standard one.

#### Changelog

:cl:  Identification
rscadd: Adds a syndicate mining hardsuit.  
tweak: All mining suit storage units outside the station have been altered to have the syndicate mining hardsuit rather than the standard one. 
tweak: Red cross on a donor borg skin replaced with a blue one.
tweak: The syndicate hardsuit legion drop has been replaced with the syndicate mining hardsuit.
/:cl:

### CODE REVIEW

**Comment by fluffe9911** [comment]
They are ex nanotransin employees but are not hired by the syndicate fyi

**Comment by nesisknesis** [comment]
I just checked the code and it does not mention this so checkmate  😎

**Comment by HomocidlePancakes** [comment]
Green, purple and brown. Are you drunk?

**Comment by nesisknesis** [comment]
What?

**Comment by HomocidlePancakes** [comment]
It's fugly. Especially the green eyes

**Comment by nesisknesis** [comment]
Ack, okay, ill see what I can do when I have time.

**Comment by Altoids1** [comment]
It's not *fugly* by any means. I don't really see too much of a problem with it, although I will say it probably will be hard to make it obvious that this is a free miner suit and not a genuine antag syndicate suit variant or something. That might cause some accidental causes of validing the freeminers, which is sad.

I like this PR though. Freeminers (and other outer-spessmen) having their own sorta suit is really nifty.

**Comment by HomocidlePancakes** [comment]
It is *super fugly*. Looks like someone decapitated nukie suit and attached it's head to the mining hardsuit even though they don't complement each other in any way and that makes it scuffed AF. Pretty sure the second picture is supposed to be combat mode but it's the same sprite with sligtly dimed eyes and thats fugly too.

**Comment by nesisknesis** [comment]
Its actually the same sprite. The colors are the same as the mining one, and without the tan it looked strikingly similar to the engineering hardsuit, which also has similar line work to the syndi hardsuit. There is no combat mode. If you're so triggered by it then what color scheme would make it look better.

**Comment by HomocidlePancakes** [comment]
Helmet is camouflaged and the suit is not. Not to mention that they have diffrent colors making them up. You've decapitated Barbie and attached her head to the body of Ken.

**Comment by nesisknesis** [comment]
Refined the default helmet by looking at an old sprite without the light and removed the tan colour.
image

image

**Comment by HomocidlePancakes** [comment]
Looks good now. Can you show the side sprite?

**Comment by Oakboscage** [comment]
This is just bad tbh.

**Comment by HomocidlePancakes** [comment]
It’s gonna get merged anyway

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm [hunk 1]
```diff
@@ -349,7 +349,7 @@
 			mask = /obj/item/clothing/mask/breath
 		if("Operative")
 			id_job = "Operative"
-			outfit = /datum/outfit/syndicatecommandocorpse
+			outfit = /datum/outfit/syndicateminercorpse //yogs - no more 8 TC hardsuit for killing a lavaland mob
```
Comment (HomocidlePancakes) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm:352]: Is that an unmentioned change? Excuse me what the fuck!?

Comment (nesisknesis) [reply code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm:352]: No, this is mentioned in the changelog. Lowest tweak.

**Comment by nesisknesis** [comment]
Will be closing this because I can think of something more interesting rather than brown syndi suit

**Pull request closed without merging**